server.js

```diff
@@ -0,0 +1,13 @@
+const server = require('express')();
+
+const dbRouter = require('./data/db-router.js');
+
+server.use(require('express').json());
+
+server.use('/api/posts', dbRouter);
+
+server.get('/', (req, res) => {
+  res.send('Server running...');
```

Comment:
can't underestimate the importance of sanity checks. always start out with one of these